vector/src/main/java/im/vector/app/features/home/room/list/home/HomeRoomListViewModel.kt

```diff
@@ -346,21 +339,28 @@ class HomeRoomListViewModel @AssistedInject constructor(
     }
 
     override fun onCleared() {
+        filteredPagedRoomSummariesLive?.livePagedList?.let { livePagedList ->
+            _roomsLivePagedList.removeSource(livePagedList)
+        }
         super.onCleared()
-        filteredPagedRoomSummariesLive?.livePagedList?.removeObserver(internalPagedListObserver)
```

Comment:
No need to call this anymore?

Reply:
filteredPagedRoomSummariesLive?.livePagedList?.let { livePagedList ->
            _roomsLivePagedList.removeSource(livePagedList)
        }
This replaces the observer ^